src/applications/disability-benefits/all-claims/utils/form0781.js

```diff
@@ -0,0 +1,22 @@
+// All flippers for the 0781 Papersync should be added to this file
+import { isClaimingNew } from '.';
+
+/**
+ * Checks if the modern 0781 flow should be shown if the flipper is active for this veteran
+ * All 0781 page-specific flippers should include a check against this top level flipper
+ *
+ * @returns
+ *   TRUE if
+ *     - is set on form via the backend
+ *     - Veteran is claiming a new disability
+ *     - Veteran has selected connected condition choices on 'screener page'
+ *   else
+ *     - returns false
+ */
+export function showForm0781Pages(formData) {
+  return (
+    formData?.syncModern0781Flow === true &&
+    isClaimingNew(formData) &&
+    formData.mentalHealth.conditions?.none !== true
```

Comment:
Wouldn't also at least one condition, other than 'none', have to be true?

Reply:
yes, you are correct. I didn't check if the page would allow users to select _nothing at all_ on the screener page. I will update

Reply:
Updated. now it will check for any condition (key)  that is not `none` that has a value of `true`